controller/src/main/java/org/jboss/as/controller/persistence/ConfigurationFile.java

```diff
@@ -763,6 +763,16 @@ private File mkdir(final File dir) {
         return dir;
     }
 
+    private static boolean forcedMove(Path from, Path to) {
+        try {
+            Files.move(from, to);
```

Comment:
If you call method forceMove maybe it should use StandardCopyOption.REPLACE_EXISTING